audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/model/AuditErrorMessage.java

```diff
@@ -109,23 +109,23 @@ public static IUserId create() {
         }
 
         /**
-         * Adds the user context information to the message.
+         * Adds the context information to the message.
          */
         public interface IContext {
             /**
-             * Adds the user context information.
+             * Adds the context information.
              *
-             * @param context user context for the request.
+             * @param context the context for the request.
```

Comment:
```suggestion
             * @param context , the reason why the user wants access to the data
```
Something a bit more descriptive of what context actually is

Reply:
Updated